### PR TITLE
add release volume info into report [com1]

### PR DESCRIPTION
**Pull request by awirb**
* update info in log for release volume used to pick friction model - add info that it is based on the grid resolution

### CODE REVIEW

File: avaframe/com1DFA/com1DFA.py [hunk 1]
```diff
@@ -2607,15 +2609,16 @@ def prepareVarSimDict(standardCfg, inputSimFiles, variationDict, simNameExisting
         # if frictModel is samosATAuto compute release vol
         if cfgSim["GENERAL"]["frictModel"].lower() == "samosatauto":
             pathToDemFull = pathlib.Path(cfgSim["GENERAL"]["avalancheDir"], "Inputs", pathToDem)
-            relVolume = fetchRelVolume(rel, cfgSim, pathToDemFull, inputSimFiles["secondaryReleaseFile"])
+            relVolume, relDEMSize = fetchRelVolume(rel, cfgSim, pathToDemFull, inputSimFiles["secondaryReleaseFile"])
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:2612]: see note above about relDEMSize

File: docs/com1DFAAlgorithm.rst [hunk 2]
```diff
@@ -436,6 +436,17 @@ The mesh values are updated with the particles properties using
...
 
+Compute pressure
+~~~~~~~~~~~~~~~~~
+Pressure is computed following:
+
+.. math::
+
+  p = \mathbf{\rho}  \mathbf{u}²,
+
+where :math:`\rho` is the snow density and :math:`\mathbf{u}` is the flow velocity.
```
Comment (fso42) [inline docs/com1DFAAlgorithm.rst:447]: Just an idea: give the name of the code function? I.e. computePressuer in DFAFunctionsCython?

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/checkCfg.py [hunk 3]
```diff
@@ -65,7 +65,7 @@ def checkCellSizeKernelRadius(cfg):
     return cfg
 
 
-def checkCfgFrictionModel(cfg, relVolume=''):
+def checkCfgFrictionModel(cfg, relVolume='', relDEMSize=''):
```
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/com1DFA/checkCfg.py:68]: done

File: docs/com1DFAAlgorithm.rst (hunk 2 above)
Comment (awirb) [reply docs/com1DFAAlgorithm.rst:447]: added

**Review by fso42 — APPROVED** [review]

**Pull request merged**